### ISSUE TITLE
PARSE_ERROR when function argument list has trailing comma

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#365]
A function declaration, where the argument list has a trailing comma, such as:

```
public function foo(
    required any bar, 
    required any baz,
) {}
```

will trigger a `PARSE_ERROR`. The code works fine (in Railo 4.2 at least).

**Comment by KamasamaK** [comment]
I don't believe this works in ACF. Must just be Railo/Lucee.

**Comment by TheRealAgentK** [comment]
If it doesn't work in ACF, then we should currently not go out of our way to support it. As much as I'd like to support Lucee properly at some point, let's focus on getting 1.2 and 1.3 out and then have a proper think about refactoring certain parts for 2.0 - and supporting Lucee properly could well be one of those things.

**Comment by TheRealAgentK** [comment]
Needs to be looked at in cfparser.

**Comment by ryaneberly** [comment]
this is resolved.  will be included in v 1.3.0
